babel/numbers.py

```diff
@@ -561,6 +563,18 @@ def __init__(self, pattern, prefix, suffix, grouping,
         else:
             self.scale = 0
 
+    def __copy__(self):
```

Comment:
Uhhhh... you know, you're right. I can't remember why I was ~sure~ that copying didn't work. :|